src/main/java/com/base64/d/Application.java

```diff
@@ -0,0 +1,9 @@
+package com.base64.d;
+
+public class Application {
```

Comment:
Renamed the `main` class to `Application`. Two reasons:
1. Java classes should be capitalized (and have camel casing, which you have on the rest of your files).
2. The `main` method is a very specific method in Java, so that is kind of a reserved name (not literally a "reserved keyword" in the Java language, but more through convention).